src/video/opengl/shaders.c

```diff
@@ -128,6 +128,11 @@ void bind_uniform_4fv(GLuint program_id, const char *name, GLfloat *data) {
     glUniformMatrix4fv(ref, 1, GL_FALSE, data);
 }
 
+void bind_uniform_block(GLuint program_id, const char *name, GLuint binding) {
+    GLuint ref = glGetUniformBlockIndex(program_id, name);
+    glUniformBlockBinding(program_id, ref, binding);
```

Comment:
Check `ref` before use. If something went wrong and the block is not there you will use -1 as unfirom block index and unpredictable stuff might happen.

Reply:
Good to know. Checks added and one type fixed.